classes/pdf/HTMLTemplateDeliverySlip.php

```diff
@@ -66,7 +66,7 @@ public function __construct(OrderInvoice $order_invoice, $smarty)
     public function getHeader()
     {
         $this->assignCommonHeaderData();
-        $this->smarty->assign(array('header' => $this->l('Delivery')));
+        $this->smarty->assign(array('header' => HTMLTemplateInvoice::l('Delivery')));
 
```

Comment:
Hi,

thank you for reporting, it will be better to user HTMLTemplateDeliverySlip::l as this function is static in HTMLTemplate overrides by HTMLTemplateDeliverySlip

Best regards